DAI/operations/DistanceOperations.py

```diff
@@ -0,0 +1,20 @@
+import numpy as np
+from interfaces.interfaces import BoundingBox
+
+def calculate_object_distance(lidar_points_2d: np.array, bounding_box: BoundingBox):
```

Comment:
Add a function output type hinting.
```suggestion
def calculate_object_distance(lidar_points_2d: np.array, bounding_box: BoundingBox) ->np.float:
```

Reply:
Additionally also return the coordinate of the brightest pixel value, this will help with debugging.